src/program.js

```diff
@@ -67,10 +67,14 @@ export class Program {
         if (cmd) {
           prefix = `${cmd}: `;
         }
-        console.error(prefix + error.toString());
+
+        console.error();
```

Comment:
These empty ```console.error()``` method calls  look strange, I suppose that they are used to create empty lines, 
how about prefixing the interpolated strings with a ```\n```? 
e.g. ```console.error(`\n${prefix}${error.stack}`);```